samples/client-provider/query.fsx

```diff
@@ -8,7 +8,1416 @@ open System.Collections.Generic
 Fable.Core.JsInterop.importAll<unit> "isomorphic-fetch"
 #endif
 
-type MyClient = GraphQLProvider<"http://localhost:8083">
+// Introspection response from the server
+// This can be optionally used as a second parameter to the Provider
+// If using this as a second parameter, connection to the server is not needed to
+// compile the schema and have IntelliSense features when developing
+[<Literal>]
+let introspection = """{
```

Comment:
I think maybe we should use another type provider to load the introspection from a text file. This avoids having this enormous introspection schema as a string in the code.

Reply:
Maybe not another type provider, but it can be an optional parmeter of an existing one.

Reply:
All right, I'll be working on something for that.